store/userStore.ts

```diff
@@ -0,0 +1,2 @@
+import { create } from 'zustand';
+
```

Comment:
Need to add the base for the store. https://medium.com/globant/react-state-management-b0c81e0cbbf3#7c3b Please look at the ending of the file too. How they export the store.

const dataStore = (set, get) => ({
  example: [],
});

Reply:
I would not name this userStore. It's not used just for the user. It's used to store user, weekly picks, nfl teams, etc. This is used for everything.